automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala

```diff
@@ -35,6 +38,22 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cl
           val status = billingPage.waitForCreateCompleted(billingProjectName)
           withClue(s"Creating billing project: $billingProjectName") { status shouldEqual "success" }
         }
+
+        //note that this next part of the test *could* be a different test on it's own, but we have chosen to
+        //build it into this test case because:
+        // a) having it in it's own test case would add another ~5 minutes to the test cycle
+        // b) this test would fail anyway if billing project creation failed
+
+        //should you decide to break this out into it's own test, billing project creation for it should
+        //be driven by API and not UI
+
+        val originalBillingAccount = Google.billing.getBillingProjectAccount(billingProjectName)
+        assertResult(Some(Config.Projects.billingAccountId))(originalBillingAccount)
```

Comment:
These tests have been using `should` matchers, i.e.:
```
originalBillingAccount shouldBe Some(Config.Projects.billingAccountId)
```
In other cases, there's some extra power in the `should` matchers that you don't have with `assertResult`, but I guess it doesn't matter in this case.